charts/portworx/v2.8.1/templates/portworx-csi.yaml

```diff
@@ -65,32 +116,70 @@ spec:
       containers:
         - name: csi-external-provisioner
           imagePullPolicy: Always
-          image: "{{ template "px.getcsiProvisioner" . }}/csi-provisioner:v0.2.0"
+          image: {{ template "px.getCSIProvisionerImage" . }}
+          args:
+            - "--v=3"
+            {{- if (semverCompare "< 1.19.0" .Capabilities.KubeVersion.GitVersion) }}
+            - "--provisioner=pxd.portworx.com"
+            - "--enable-leader-election"
+            {{- end }}
+            - "--csi-address=$(ADDRESS)"
+            {{- if and (semverCompare ">=1.13.0" .Capabilities.KubeVersion.GitVersion) (semverCompare "<1.14.0" .Capabilities.KubeVersion.GitVersion) }}
+            - "--leader-election-type=leases"
+            {{- end }}
```

Comment:
These args need to match the following:
https://github.com/portworx/px-installer/blob/master/px-spec-websvc/templates/template_portworx.gtpl#L717-L734

For example, `"--leader-election=true"` is only in v2/v3+. 

For v1 provisioner, the flag is `"--enable-leader-election"` with `"--leader-election-type=leases"`